mne/epochs.py

```diff
@@ -2073,10 +2073,11 @@ def equalize_event_counts(self, event_ids=None, method='mintime'):
         # actually remove the indices
         return self, indices
 
+    @verbose
     @fill_doc
```

Comment:
`@verbose` does a `@fill_doc` under the hood so you can remove this
```suggestion
    @verbose
```